helm/memcached-operator/helm-charts/memcached/README.md

```diff
@@ -129,4 +128,18 @@ spec:
 
 Once you've done this, you can upgrade to 3.x with Helm as normal.
 
-
+If you want prometheus-operator scrap all serviceMonitors in your cluster you need to set:
```

Comment:
If you want prometheus-operator to scrape all....

Reply:
It came from the helm chart is not something that we added.